config/locales/en.yml

```
@@ -361,10 +362,14 @@ en:
     destroy:
       removed_notice: "%{owner_name} was removed from the owners successfully"
       failed_notice: Can't remove the only owner of the gem
-    mfa_required: The gem has MFA requirement enabled, please setup MFA on your account.
+    mfa_required: The gem has multi-factor authentication requirement enabled, please setup multi-factor authentication on your account.
   settings:
     edit:
       title: Edit settings
+      webauthn_credentials: Security device
+      no_webauthn_credentials: You don't have any security devices
+      webauthn_credential_note: A security device can be can be any device that complies with the FIDO2 standard such as security and biometric keys. This device currently can only be used for web login and is therefore not a replacement for OTP based Multi-factor Authentication.
```

```suggestion
      webauthn_credential_note: A security device can be can be any device that complies with the FIDO2 standard such as security and biometric keys. Once enabled, devices currently can only be used for web login and is therefore not a replacement for OTP based multi-factor authentication.
```

> Once enabled, devices currently can only be used for web login and is therefore not a replacement for OTP based Multi-factor Authentication.

I think this is an important note that can be lost in the description. Is there a way to make this apparent to the user (eg. bold)?